src/vs/workbench/services/extensionManagement/browser/webExtensionsScannerService.ts

```diff
@@ -829,6 +834,11 @@ export class WebExtensionsScannerService extends Disposable implements IWebExten
 					webExtension.defaultManifestTranslations = null;
 				}
 			}
+			const migratedLocation = migratePlatformSpecificExtensionGalleryResourceURL(webExtension.location, TargetPlatform.WEB);
+			if (migratedLocation) {
+				update = true;
+				webExtension.location = migratedLocation;
+			}
 			return webExtension;
 		}));
 		if (update) {
```

Comment:
What if one of the web extension promises throws? That means we will never reach this code. Should there be some sort of `finally` handling here?

Reply:
There is a try/catch for each promise to log the error.